target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8064.dtsi

```diff
@@ -573,10 +573,10 @@
 			#size-cells = <1>;
 			status = "okay";
 			tsens_calib: calib@400 {
-				reg = <0x400 0x10>;
+				reg = <0x400 11>;
 			};
 			tsens_backup: backup@410 {
-				reg = <0x410 0x10>;
+				reg = <0x410 11>;
```

Comment:
What about using hex notation?

Reply:
Changed to hex but i still think 11 better explain it...

If we change to hex here should we change also speedbin to 0x4 ?